packages/forms-web-app/__tests__/unit/controllers/common/your-email-address.test.js

```diff
@@ -40,27 +43,21 @@ describe('controllers/full-appeal/submit-appeal/enter-code', () => {
 			await returnedFunction(req, res);
 			expect(res.redirect).toBeCalledWith(`/${lpaViews.ENTER_CODE}/${testId}`);
 		});
-		it('ERRORS /controllers/common/getYourEmailAddress.js', async () => {
+		it('/controllers/common/postYourEmailAddress.js valid email address', async () => {
 			const testId = 'd1f31fc7-0152-4796-ab48-52adcdd95066';
 			const testEmail = 'iamnoone@@planninginspectorate.gov.uk';
```

Comment:
Only feedback is that if we can run any tests against the @example.com domain just as a safe guard that we never accidentally write tests that fire off emails to a domain with a live mail server, otherwise LGTM